small-talk/pages/api/login.js

```diff
@@ -1,6 +1,6 @@
 import Mongoboi from "@/db/mongo"
 import  {createCookieHash}  from "@/util/smolCwypto"
-var uri = "mongodb+srv://smt_root:pokemonwithguns@smalltalkcluster0.jo4jne6.mongodb.net/?retryWrites=true&w=majority"
+var uri = "mongodb+srv://"+ "smt_root" + ":" + "pokemonwithguns" + "@smalltalkcluster0.jo4jne6.mongodb.net/?retryWrites=true&w=majority"
```

Comment:
Was there a specific reason why we format the uri like this instead of the previous way? Just curious in case I have to implement this for anything moving forward.

Reply:
No reason other than quick and dirty testing. this is a good catch. rather than raw dogging the credentials like this, i should keep it to the environment variables

```js
let uri = `mongodb+srv://${DB_USER}:${DB_PASS}@${DB_URL}`
```

Reply:
Thanks for the info!